proposals/lip-0049.md

```diff
@@ -232,15 +236,15 @@ def apply(ccu: Transaction, ccm: CCM) -> None:
         for each module mdl for which verifyCrossChainMessage exists:
             mdl.verifyCrossChainMessage(ccu, ccm)
     except:    
+        terminateChain(ccm.sendingChainID)
```

Comment:
Why do we move this function call before the event emission here and in other places? Note that `terminateChain` sends a message (with associated event) so this event would then be emitted before.

Reply:
I wanted to be consistent with what we do in LIP53 (e.g., here: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#execution). In general, I think both orderings would be fine.

Reply:
Okay, sounds good.